website/content/docs/connect/proxies/envoy-extensions/usage/apigee-ext-authz.mdx

```diff
@@ -0,0 +1,203 @@
+---
+layout: docs
+page_title: Delegate authorization to Apigee
+description: Learn how to use the `ext-authz` Envoy extension to delegate data plane authorization requests to Apigee. 
+---
+
+# Delegate authorization to Apigee
+
+This topic describes how to use the external authorization Envoy extension to delegate data plane authorization requests to Apigee.
+
+For a more detailed walkthrough follow [this learn guide](https://github.com/hashicorp-education/learn-consul-apigee-external-authz).
+
+## Workflow
+
+Complete the following steps to use the external authorization extension:
+
+1. Deploy the `apigee-remote-service-envoy` service and register the service in Consul
+1. Configure an `EnvoyExtensions` block in a service defaults or proxy defaults configuration entry. 
+1. Apply the configuration entry.
+
+## Configure the `apigee-remote-service-envoy` service default
```

Comment:
Thanks @boruszak! very helpful, let me know how the latest commit looks.

I didn't write the full statements in workflows as headings because they we too long